pgcli/pgcompleter.py

```diff
@@ -185,6 +190,15 @@ def get_completions(self, document, complete_event, smart_completion=None):
                 tables = suggestion['tables']
                 _logger.debug("Completion column scope: %r", tables)
                 scoped_cols = self.populate_scoped_cols(tables)
+
+                if suggestion.get('drop_unique'):
+                    # drop_unique is used for 'tb11 JOIN tbl2 USING (...'
+                    # which should suggest only columns that appear in more than
+                    # one table
+                    scoped_cols = [col for (col, count)
+                                         in Counter(scoped_cols).items()
+                                           if count > 1 and col != '*']
```

Comment:
Consider `...s JOIN t USING (x) JOIN u USING (` which should suggest columns that are present in any two of s, t, u. I think set intersection would only suggest columns that are present in all three, right? So I guess we want the union of all pairwise intersections?